chain/blocksync/blocksync.go

```diff
@@ -126,7 +127,7 @@ func (bss *BlockSyncService) HandleStream(s inet.Stream) {
 	}
 
 	writeDeadline := 60 * time.Second
-	_ = s.SetDeadline(time.Now().Add(writeDeadline))
+	_ = s.SetDeadline(build.Clock.Now().Add(writeDeadline))
```

Comment:
Socket/stream deadlines should use real time as internally they use real time.

Reply:
Fixed in [`fbf6596` (#2362)](https://github.com/filecoin-project/lotus/pull/2362/commits/fbf6596f20744628212d4b6a5c92ded32ae0693f).